shell/app/modules/application/pages/deploy/deploy.tsx

```diff
@@ -150,7 +150,7 @@ const NewDeployForm = ({ curEnv, isUpdate, setCurEnv, curBranch }: IFormProps) =
           }
           return (
             <Option key={item.name} value={item.name} disabled={!!tip}>
-              <Tooltip title={tip}>{item.name}</Tooltip>
+              {item.name}
```

Comment:
what is the reason to remove tooltip?

Reply:
Because Antd4.x Select.Option comes with its own title.